src/platform/EFR32/Logging.cpp

```diff
@@ -133,10 +133,13 @@ extern "C" void efr32Log(const char * aFormat, ...)
 
     va_start(v, aFormat);
 #if EFR32_LOG_ENABLED
+    TickType_t now;
     char formattedMsg[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE];
     static_assert(sizeof(formattedMsg) > kMaxCategoryStrLen); // Greater than to at least accommodate a ending Null Character
 
-    strcpy(formattedMsg, LOG_EFR32);
+    // strcat(formattedMsg, LOG_EFR32);
+    now = xTaskGetTickCount();
+    snprintf(formattedMsg, CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE, "[%ld] %s", now, LOG_EFR32);
```

Comment:
why do we need the timestamp
this gonna be crazy verbosy for the logs 

I would prefer a config flag for this and leave it by default as it was

Reply:
I am not totally opposed to it. Chip-tool and python controller have timestamps like that and it is still readable. Yes, you can add a flag to toggle it on and off yet. But I do see the appeal for debugging some timing-related issues.

Reply:
Reverted the change for now, will add a flag to toggle once we have a better timestamp resolution on logs.

Reply:
yes chip tool is handy but RTT debug is already crowded with logs for an embedded platform

Reply:
Yeah, I will make it a patch till we have a consensus. I feel remote scenarios might find this useful. My recommendation is to replace the `<tag >` with `<T>` and include timestamps instead.